### PR TITLE
Cleanup code in the objects, use constructor instead of calling setNa…

### PR DESCRIPTION
**Pull request by Articdive**
…me().

### CODE REVIEW

File: src/com/palmergames/bukkit/towny/object/Resident.java [hunk 1]
```diff
@@ -27,38 +27,30 @@
 import java.util.Arrays;
 import java.util.List;
 
-public class Resident extends TownBlockOwner implements ResidentModes, TownyInviteReceiver{
-
+public class Resident extends TownBlockOwner implements ResidentModes, TownyInviteReceiver {
 	private List<Resident> friends = new ArrayList<>();
-	@SuppressWarnings("unused") // Feature is disabled as of MC 1.13, maybe it'll come back.
```
Comment (LlmDl) [inline src/com/palmergames/bukkit/towny/object/Resident.java]: Why did you remove this.

File: src/com/palmergames/bukkit/towny/object/Nation.java [hunk 2]
```diff
@@ -43,28 +43,27 @@
...
-		tag = "";
-        isPublic = TownySettings.getNationDefaultPublic();
-        isOpen = TownySettings.getNationDefaultOpen();
+		super(name);
 	}
 
 	public void setTag(String text) throws TownyException {
 
-		if (text.length() > 4)
+		if (text.length() > 4) {
```
Comment (LlmDl) [inline src/com/palmergames/bukkit/towny/object/Nation.java]: I thought we weren't doing this anymore.

File: src/com/palmergames/bukkit/towny/object/Nation.java [hunk 3]
```diff
@@ -43,28 +43,27 @@
 	private Town capital;
 	private double taxes, spawnCost;
 	private boolean neutral = false;
-	private String nationBoard = "/nation set board [msg]", tag;
+	private String nationBoard = "/nation set board [msg]";
```
Comment (LlmDl) [inline src/com/palmergames/bukkit/towny/object/Nation.java]: this is essentially a formatting change and not doing anything helpful

File: src/com/palmergames/bukkit/towny/object/Resident.java (hunk 1 above)
Comment (Articdive) [reply src/com/palmergames/bukkit/towny/object/Resident.java]: If it's unused and suppressed we can just remove it and use git history to check how we did it back then, this is how we do it in other cases?

File: src/com/palmergames/bukkit/towny/object/Nation.java (hunk 2 above)
Comment (Articdive) [reply src/com/palmergames/bukkit/towny/object/Nation.java]: This was before you told me not to do any formatting changes anymore, literally in the middle of changing it you told me that

File: src/com/palmergames/bukkit/towny/object/Resident.java (hunk 1 above)
Comment (LlmDl) [reply src/com/palmergames/bukkit/towny/object/Resident.java]: The only way I'll remember to readd it someday is if it is still in place.

Comment (Articdive) [reply src/com/palmergames/bukkit/towny/object/Resident.java]: I can comment it out, then it's not a variable and then we don't need a @SuppressWarnings
